### DISCUSSION TITLE
Omega CN402 connection issue

### DISCUSSION THREAD

**Discussion by mikebest66** [artisan-roaster-scope/artisan#1269] [General]
I'm currently seeking the correct configurations to establish a connection with the Omega CN402 using my Ambex YM-15 roaster. The provided software successfully reads the temperature when set to COM5 with a baud rate of 38400 and using an 8-N-1 configuration.

Initially, I followed the instructions to install the Phidgets controller. However, when it failed to detect the device, I reached out to the manufacturer. They connected me with the Logging Dynamics software designer, who informed me that the Phidgets controller isn't utilized in this particular unit. Consequently, I uninstalled the associated drivers, and the "Logging Dynamics" software began functioning correctly. However, the software designer was uncertain about the settings required for the Artisan software.

I have confirmed that the cable connection and temperature reading are functioning correctly. My question now is, how do I configure the settings within Artisan to establish communication with the Omega CN402? It appears that the Omega devices listed in Artisan do not correspond to the one I have. Any guidance you can provide would be greatly appreciated.

**Comment by MAKOMO** [comment]
The Omega CN402 is not supported by Artisan. You are welcome to implement support for this device for Artisan and send in a PR.
